web/pwa/components/MissionDetails.js

```diff
@@ -133,6 +133,21 @@ export function MissionDetails({
   const actualUserId = userId || store.userId();
   const alerts = useSnackbarAlerts();
 
+  const [
+    currentlyEmployedInCompany,
+    setCurrentlyEmployedInCompany
+  ] = React.useState(false);
+
+  React.useEffect(() => {
+    setCurrentlyEmployedInCompany(
+      mission?.company?.id &&
+        store
+          .companies()
+          .map(c => c.id)
+          .includes(mission.company?.id)
```

Comment:
:green_circle: Pas besoin vu que c'est testé juste avant
```suggestion
          .includes(mission.company.id)
```